### ISSUE TITLE
Helm errors cause all resources to show Err Applied even though they successfully deployed

### ISSUE THREAD

**Issue by waterfoul** [rancher/fleet#246]
1. Create a repo using the following yaml
```yaml
apiVersion: fleet.cattle.io/v1alpha1
kind: GitRepo
metadata:
  name: test-repo
  namespace: fleet-default
spec:
  branch: main
  insecureSkipTLSVerify: false
  paths: []
  repo: https://github.com/waterfoul/testRepo.git
  targets:
  - clusterSelector: {}
type: fleet.cattle.io.gitrepo
```

2. Open the test repo in the rancher UI
3. Observe: All resources have "Err Applied" in the state column (this is unexpected)
4. Go to Bundles
5. Observe: test-repo and test-repo-working are listed as active, test-repo broken is listed as Err Applied (this is expected)
6. Go to Cluster Explorer for the working-namespace
7. Observe: the nginx deployment is successfully deployed (this is expected)
7. Observe: the broken-namespace is missing (this is unexpected but reasonable)

gz#14632

**Comment by nickgerace** [comment]
LoE will be assigned once reproduced.

**Comment by kkaempf** [comment]
Verified fixed in 0.6.0
